### PR TITLE
fixed https://github.com/kanboard/kanboard/issues/2848

### PR DESCRIPTION
**Pull request by nepomuc**
fixed https://github.com/kanboard/kanboard/issues/2848 and also added a title attribute to the a-tag so the user can read the full filename by hovering.

test 1 - test - google chrome 2016-11-29 17 32 25

### CODE REVIEW

File: assets/js/src/Popover.js [hunk 1]
```diff
@@ -13,6 +13,10 @@ Kanboard.Popover.prototype.listen = function() {
         self.close(e);
     });
 
+    $(document).on("click", "#popover-container", function(e) {
+        self.close(e);
```
Comment (fguillot) [inline assets/js/src/Popover.js]: Can you remove this part for now? I prefer to put back this feature only if we introduce an alert box for modal box that contains a form.

Thanks

Comment (nepomuc) [inline assets/js/src/Popover.js]: I already built something to address this topic @ https://github.com/kanboard/kanboard/issues/2847

**Comment by fguillot** [comment]
Is it possible to update your pull-request without the Javascript changes?

Thank you

**Comment by fguillot** [comment]
Commit #e244c9c is merged.

**Pull request closed without merging**